ts/src/base/Exchange.ts

```diff
@@ -1818,6 +1818,13 @@ export default class Exchange {
         throw new NotSupported (this.id + ' handleDelta not supported yet');
     }
 
+    handleDeltasWithKeys (bookSide, deltas, priceKey: IndexType = 0, amountKey: IndexType = 1, countOrIdKey: IndexType = 2) {
```

Comment:
this is also very generic method and can be applied to many exchange, without need to copy and repeat hardcoded methods inside many excahnges' implementations